sdk/src/main/java/io/opentelemetry/sdk/metrics/AbstractInstrument.java

```diff
@@ -92,21 +62,12 @@ public boolean equals(Object o) {
 
     AbstractInstrument that = (AbstractInstrument) o;
 
-    return name.equals(that.name)
-        && description.equals(that.description)
-        && unit.equals(that.unit)
-        && constantLabels.equals(that.constantLabels)
-        && labelKeys.equals(that.labelKeys);
+    return descriptor.equals(that.descriptor);
```

Comment:
should this include the library info, so two libraries could have the same instrument descriptor, and not collide?

Reply:
The instrument descriptor has to be unique inside a MeterSdk which is uniquely identified by the LibraryInfo, so I don't think we should include the LibraryInfo

Reply:
ah, yes. that makes sense.  👍